aws/resource_aws_ecs_service_test.go

```diff
@@ -2092,6 +2092,7 @@ resource "aws_ecs_service" "ghost" {
   cluster = "${aws_ecs_cluster.default.id}"
   task_definition = "${aws_ecs_task_definition.ghost.family}:${aws_ecs_task_definition.ghost.revision}"
   scheduling_strategy = "DAEMON"
+  deployment_minimum_healthy_percent = "50"
```

Comment:
We should implement this in a separate acceptance test and test configuration to ensure both the unconfigured and configured cases either work or are covered by `ExpectError` testing as appropriate. Copy-pasting most of the existing test is fine. 👍

Reply:
Having an acceptance test for daemon that does not set `deployment_minimum_healthy_percent` to something less than 100 results in:
```
InvalidParameterException: Both maximumPercent and minimumHealthyPercent cannot be 100 as this will block deployments
```